go/stats/main.go

```diff
@@ -1764,3 +1781,38 @@ var Verb = TypeName{
 		},
 	},
 }
+
+// Detect the vip on management
+func detectVIP(management pfconfigdriver.ManagementNetwork) {
+	var keyConfCluster pfconfigdriver.NetInterface
+	keyConfCluster.PfconfigNS = "config::Pf(CLUSTER," + pfconfigdriver.FindClusterName(ctx) + ")"
+
+	keyConfCluster.PfconfigHashNS = "interface " + management.Int
+	pfconfigdriver.FetchDecodeSocket(ctx, &keyConfCluster)
+	// Nothing in keyConfCluster.Ip so we are not in cluster mode
+	if keyConfCluster.Ip == "" {
+		VIP[management.Int] = true
+		return
+	}
+
+	eth, _ := net.InterfaceByName(management.Int)
+	adresses, _ := eth.Addrs()
+	var found bool
+	found = false
+	for _, adresse := range adresses {
```

Comment:
This `for` loop appears to iterate over all `adresses` even after `found=true`. Is a `return` statement needed right after `found=true`?

Would it be faster to define `VIP[management.Int] = false` at the beginning of this function, and only set `VIP[management.Int] = true` and `return` immediately if `IP.Equal(VIPIp[management.Int])` ?